sqlmesh/core/model/seed.py

```diff
@@ -29,7 +29,7 @@ def columns_to_types(self) -> t.Dict[str, exp.DataType]:
     def column_hashes(self) -> t.Dict[str, str]:
         df = self._get_df()
         return {
-            column_name: str(zlib.crc32(df[column_name].to_json().encode("utf-8")))
+            column_name: str(zlib.crc32(df[column_name].to_json().encode("utf-8")))  # type: ignore
```

Comment:
this didn't raise for me, why is it for you?

Reply:
have no idea TBH. Gonna find out

Reply:
Should've upgraded `pandas-stubs`.